package.json

```diff
@@ -12,10 +12,10 @@
     "lint": "eslint --ext .ts,.tsx,.js,.jsx src/ perf/",
     "doc": "typedoc src/mod.ts",
     "build": "rollup --config rollup.config.js",
-    "build-dev": "DEV=1 rollup --config rollup.config.js",
+    "build-all": "rollup --config rollup.config.js && DEV=1 rollup --config rollup.config.js && CJS=1 rollup --config rollup.config.js && CJS=1 DEV=1 rollup --config rollup.config.js",
```

Comment:
out of curiosity, why do we use rollup instead of letting `tsc` generate one module type or another?

Reply:
tsc does not create a single file. It does not copy over the `wasm/{debug,release}/replicache_client.js` file